### PR TITLE
[dashboard] Fixes

### PR DESCRIPTION
**Pull request by kristofferj**

### CODE REVIEW

File: packages/@sanity/dashboard/src/widget.css [hunk 1]
```diff
@@ -45,6 +44,7 @@
   min-height: 4em;
   height: 4em;
   margin-top: auto;
+  overflow:hidden;
```
Comment (thomax) [inline packages/@sanity/dashboard/src/widget.css]: Is there no linter in operation which auto-adds a whitespace after `:`?

Comment (kristofferj) [reply packages/@sanity/dashboard/src/widget.css]: oh. i will add that space manually

Comment (kristofferj) [reply packages/@sanity/dashboard/src/widget.css]: There is no auto-fix enabled. But stylelint gave it a red line that i missed

**Comment by kristofferj** [comment]
Yes, this fix "wrong number of feed items".

**Pull request merged**